### PR TITLE
allow cors request from react fetch

### PR DESCRIPTION
**Pull request by LukeSchlangen**
I followed this tutorial and ran into errors with `.json` not being able to parse and `cors` being prevented because the client side wasn't allowing it (even though the server is allowing it).

I hope this is the right way to make a PR. I did it from clicking the links, so I apologize for the branch name being `patch-1`

### CODE REVIEW

File: docs/docs/ecommerce-tutorial/index.md [hunk 1]
```diff
@@ -194,7 +195,7 @@ const Checkout = class extends React.Component {
             console.log("Transaction processed successfully")
             this.resetButton()
             this.setState({ paymentMessage: "Payment Successful!" })
-            return res.json()
+            return res
```
Comment (pieh) [inline docs/docs/ecommerce-tutorial/index.md]: Is this needed too?

**Comment by pieh** [comment]
> I hope this is the right way to make a PR. I did it from clicking the links, so I apologize for the branch name being `patch-1`

Don't worry about that - branch name suggestion is for you, so you can navigate your fork branches more easily - it doesn't affect us

File: docs/docs/ecommerce-tutorial/index.md (hunk 1 above)
Comment (LukeSchlangen) [reply docs/docs/ecommerce-tutorial/index.md]: I did need to do that in order to get it to work. I was getting a parsing error when I left it in.

**Comment by pieh** [comment]
@gillkyle did you work on e-commerce tutorial? If so I would appreciate your review of this PR

**Comment by LukeSchlangen** [comment]
Thank you! I love Gatsby! Happy to help even a little bit!

**Pull request merged**